tests/js/spec/utils/useApi.spec.tsx

```diff
@@ -0,0 +1,58 @@
+import {mountWithTheme} from 'sentry-test/reactTestingLibrary';
+
+import {Client} from 'app/api';
+import useApi from 'app/utils/useApi';
+
+describe('useApi', function () {
+  let apiInstance: Client;
+
+  type Props = {
+    /**
+     * Test passthrough API clients
+     */
+    api?: Client;
+    /**
+     * Test persistInFlight
+     */
+    persistInFlight?: boolean;
+  };
+
+  const MyComponent: React.FC<Props> = ({api, persistInFlight}) => {
+    apiInstance = useApi({api, persistInFlight});
+    return <div />;
+  };
```

Comment:
not sure.. but should we move this out of `describe`?

Reply:
I'm indifferent. I think we usually have stuff in `describe` blocks